app/AppKernel.php

```diff
@@ -23,6 +23,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
(related opinion of mine: even if it's a demo app, it should still not have an SQLite database file in the repo and instead have a one-line instruction on how to populate it, but that's a different discussion :p)